### ISSUE TITLE
Teleporter Importer UI Failure

### ISSUE THREAD

**Issue by cpriest** [orientechnologies/orientdb-studio#512]
Trying to use the new Teleporter UI in studio, when I get to the Step 3: Filters page, there are repeated JS errors:

 - This error repeats about twice per second.
 - I have verified connectivity from the Step 1 page
 - If I try and go [prev] and [next] then the page does not even render correctly (only way to reattempt is to refresh, log back in and start from Step 1)

**Here is the stack trace:**
```
TypeError: this._trackByFn is not a function
    at DefaultIterableDiffer.check (main.js:20931)
    at DefaultIterableDiffer.diff (main.js:20904)
    at DualListComponent.ngDoCheck (main.js:182326)
    at checkAndUpdateDirectiveInline (main.js:24905)
    at checkAndUpdateNodeInline (main.js:26400)
    at checkAndUpdateNode (main.js:26343)
    at debugCheckAndUpdateNode (main.js:27200)
    at debugCheckDirectivesFn (main.js:27141)
    at Object.eval [as updateDirectives] (VM1606 TeleporterComponent.ngfactory.js:1319)
    at Object.debugUpdateDirectives [as updateDirectives] (main.js:27126)
    at checkAndUpdateView (main.js:26310)
    at callViewAction (main.js:26658)
    at execEmbeddedViewsAction (main.js:26616)
    at checkAndUpdateView (main.js:26311)
    at callViewAction (main.js:26658)
```
 _ | _ | _ | _ 
-- | -- | -- | --
  | (anonymous) | @ | main.js:159549
  | (anonymous) | @ | main.js:156072
  | $digest | @ | main.js:163212
  | (anonymous) | @ | main.js:30594
  | schedulerFn | @ | main.js:17706
  | SafeSubscriber.__tryOrUnsub | @ | main.js:45257
  | SafeSubscriber.next | @ | main.js:45204
  | Subscriber._next | @ | main.js:45144
  | Subscriber.next | @ | main.js:45108
  | Subject.next | @ | main.js:65987
  | EventEmitter.emit | @ | main.js:17680
  | checkStable | @ | main.js:17914
  | onHasTask | @ | main.js:17962
  | ZoneDelegate.hasTask | @ | main.js:257344
  | ZoneDelegate._updateTaskCount | @ | main.js:257361
  | Zone._updateTaskCount | @ | main.js:257192
  | Zone.runTask | @ | main.js:257147
  | drainMicroTaskQueue | @ | main.js:257509
  | ZoneTask.invoke | @ | main.js:257387
  | load (async) |   |  
  | invokeAddFunc | @ | main.js:258302
  | scheduleEventListener | @ | main.js:258329
  | ZoneDelegate.scheduleTask | @ | main.js:257316
  | onScheduleTask | @ | main.js:257204
  | ZoneDelegate.scheduleTask | @ | main.js:257310
  | Zone.scheduleTask | @ | main.js:257159
  | Zone.scheduleEventTask | @ | main.js:257176
  | zoneAwareAddListener | @ | main.js:258374
  | addEventListener | @ | VM1565:3
  | (anonymous) | @ | main.js:42196
  | Observable._trySubscribe | @ | main.js:30915
  | Observable.subscribe | @ | main.js:30903
  | (anonymous) | @ | main.js:31029
  | ZoneAwarePromise | @ | main.js:257692
  | Observable.toPromise | @ | main.js:31027
  | TeleporterService.getTablesNames | @ | main.js:182117
  | TeleporterComponent.getTablesNames | @ | main.js:181044
  | TeleporterComponent.switchConfigStep | @ | main.js:181024
  | (anonymous) | @ | VM1606 TeleporterCom…nt.ngfactory.js:751
  | handleEvent | @ | main.js:26057
  | callWithDebugContext | @ | main.js:27526
  | debugHandleEvent | @ | main.js:27114
  | dispatchEvent | @ | main.js:22673
  | (anonymous) | @ | main.js:23287
  | (anonymous) | @ | main.js:52930
  | ZoneDelegate.invokeTask | @ | main.js:257330
  | onInvokeTask | @ | main.js:17940
  | ZoneDelegate.invokeTask | @ | main.js:257329
  | Zone.runTask | @ | main.js:257129
  | ZoneTask.invoke | @ | main.js:257383

**Comment by wolf4ood** [comment]
Hi @cpriest 

which version of OrientDB?

**Comment by cpriest** [comment]
3.0.1

**Comment by wolf4ood** [comment]
@cpriest 

ok let me check

**Comment by wolf4ood** [comment]
Hi @cpriest 

i've just pushed a fix. It will be available in the next GA release 3.0.3.

Thanks

**Comment by cpriest** [comment]
Great, thanks!
